R/timeseries.R

```diff
@@ -0,0 +1,20 @@
+#' time.series.data
+#'
+#' time.series.data
+#' 
+#' @param data.element character name of data element within available categories by year for state
+#' @param years vector of integers specifying all years available for state
```

Comment:
Need argument to adjust y axis. Idea is `y.scale` with a default = NA that lets R auto scale.